realm/realm-library/src/objectServer/java/io/realm/RealmFunctions.java

```diff
@@ -15,16 +15,30 @@
  */
 package io.realm;
 
-import org.bson.BsonValue;
+import org.bson.codecs.configuration.CodecRegistry;
 
 import io.realm.internal.jni.JniBsonProtocol;
 
-public class RealmFunctions {
+// FIXME This class is only a placeholder for JNI round trip as  until actual RealmFunctions
```

Comment:
It is fine to keep this public as long as there is a Javadoc stating it is still work-in-progress. The `v10` branch is considered under development anyway, so merging "half-finshed" work should be okay as long as it is marked as such and doesn't break existing functionality.